Build system changed in 2024.86

Building from `2022.83` to `2024.86` , I had to do some changes to fix the build system.
Just pointing this out, as I was facing this error:

```
configure: error: cannot find install-sh, install.sh, or shtool in "." "./.." "./../.."
```

The fix done to build is this:

```diff
 configure_package() {
@@ -13,11 +13,19 @@ configure_package() {
 
        CC=${BUILD_CC} LDFLAGS="${BUILD_LDFLAGS}" ./configure \
                 --build=${MACHTYPE} --host=${BUILD_TARGET} --target=${BUILD_TARGET} --prefix=${INSTALL_PREFIX} \
+                --srcdir=src \
                 --disable-zlib
 }
 
+premake_package() {
+       echo_notice "Patching config"
+       sed -i 's!src/src!src!g' Makefile
+       ln -sv ../libtommath src/libtommath
+       ln -sv ../libtomcrypt src/libtomcrypt
+}
```

--- a/configure.ac
+++ b/configure.ac
@@ -5,7 +5,7 @@
 # of the platform checks have been taken straight from OpenSSH's configure.ac
 # Huge thanks to them for dealing with the horrible platform-specifics :)
 
-AC_PREREQ([2.59])
+AC_PREREQ([2.70])
 AC_INIT
 
 ORIGCFLAGS="$CFLAGS"

